README.md

```diff
@@ -20,7 +20,7 @@ This action currently supports these versions of MRI, JRuby and TruffleRuby:
 | `truffleruby` | 19.3.0 - 24.0.1, head |
 | `truffleruby+graalvm` | 21.2.0 - 24.0.1, head |
 
-`ruby-debug` is the same as `ruby-head` but with assertions enabled (`-DRUBY_DEBUG=1`).  
+`ruby-debug` is the same as `ruby-head` but with assertions enabled (`-DRUBY_DEBUG=1`).
```

Comment:
I don't see any difference in rendered output either on github.com or using Markdown preview in vscode.

What difference do you see this trailing whitespace causing?

Reply:
This specific one does not matter, because there is a blank line below.

But for instance for `This caching speeds up installing gems significantly and avoids too many requests to RubyGems.org.` it matters, https://github.com/ruby/setup-ruby/blob/f9fb0e0deb0c52f36e8aa8d16c1edfb652b10093/README.md vs https://github.com/ruby/setup-ruby

Reply:
Honestly, an actual paragraph reads far more easily than a list of sentences that aren't really bullet points, but ok...